active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,575 @@
+
+A teaching tip is a content-rich nonmodal flyout that provides contextual information. A teaching tip looks like a flyout, except that it may target a UI element with a tail or be persistent and require the user to manually dismiss it. A teaching tip is often used for informing, reminding, and teaching users about important and new features that may enhance their experience. 
+
+**Important APIs:** [TeachingTip class]() [Link TODO]
+
+A teaching tip may also be light-dismiss enabled or used without a tail indicating reference to any specific UI element. 
+
+
+### Is this the right control? 
+
+Use a **TeachingTip** control to inform users about new and important updates and features, focus users on a component to advise them about its options or state, remind users of nonessential options that would improve their experience, or teach users how a task should be completed. 
+
+Because a teaching tip is impermanent, it would not be the recommended control for notifying users about important errors and statuses, prompting users for event handling, nor displaying a list of items or events that have been handled. 
+
+<!-- Explain why TT is not recommended for notifications --> 
+
+
+## Examples
+
+The teaching tip has several states, including these notable ones.
+
+A teaching tip can point to a specific UI element to enhance contextual clarity of the information it is presenting. 
+
+<a href="url"><img src="TeachingTipPointerSample.jpg" height="600"></a>
+
+A teaching tip can be used without a tail when the information that it is presenting is not specific to a UI element. 
+
+<a href="url"><img src="TeachingTipSampleApp.jpg" height="600"></a>
+
+A teaching tip can require manual acknowledgement before dismissing or a teaching tip may also be light-dismiss enabled so that it will dismiss when a user scrolls or interacts with other elements of the application.
+
+<a href="url"><img src="TeachingTipLightDismissSample.jpg" height="600"></a>
+
+
+## Create a teaching tip
+
+<!-- Use the Password property to get or set the contents of the PasswordBox. You can do this in the handler for the PasswordChanged event to perform validation while the user enters the password. Or, you can use another event, like a button Click, to perform validation after the user completes the text entry. -->
+
+Here's the XAML for a teaching tip control that demonstrates the default look of the TeachingTip with a title and subtitle. 
+
+When the user clicks the button, a teaching tip will appear and display a message to the user. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick"></Button>
+<muxc:TeachingTip> 
+    <muxc:TeachingTip x:Name="AutoSaveTip"
+        Title="Saving automatically"
+        Subtitle="We save your changes as you go - so you never have to.">
+    </muxc:TeachingTip>
+</muxc:TeachingTip.Attach>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
```

Comment:
By behaviors, are you refering to [this](https://github.com/Microsoft/XamlBehaviors)?

Reply:
If Behavior is referring to the XAML Behaviors noted above, I am not confident that an example of it here would maintain the scope of focus otherwise shown throughout Teaching Tip's documentation (which focuses on the features of TeachingTip as opposed to the spectrum of conditions and means for opening one) as much as if an example of Teaching Tip was instead included in the examples and documentation for Behavior? I am open to understanding this issue better if you think other or if there is a lapse of understanding on my part. 

For what its worth, we are planning on showing a least a couple of these in the XAML Sample Gallery, such as how to delay an opening, etc.

Reply:
@michael-hawker, I know there's been a lot of activity on this repo, so I just wanted to ping you again and make sure my answers are responding to the [Behavior](https://github.com/Microsoft/XamlBehaviors) you intended to surface?